sunbird_service_api_test/src/test/java/org/sunbird/common/action/CourseBatchUtil.java

```diff
@@ -11,6 +11,7 @@
 public class CourseBatchUtil {
 
   private static String openCourseBatchId = "";
+  private static String inviteOnlyCourseBatchId = "";
```

Comment:
No need to initialize, we are checking for initialization in getMethod for the variable.